deployment.yaml

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
@apuntamb, what will be the expected value for `ENABLE_RBAC` in this case?

Reply:
I will define the `ENABLE_RBAC` flag in the parameters with value `True`.